jetson/filter/src/__main__.py

```diff
@@ -164,6 +184,21 @@ def _driveVelDataCallback(self, channel, msg):
         drive_vel_data = DriveVelData.decode(msg)
         self.encoder_velocities[drive_vel_data.axis] = drive_vel_data.vel_percent
 
+    def _driveVelCmdCallback(self, channel, msg):
+        drive_vel_cmd = DriveVelCmd.decode(msg)
+        self.drive_v_left = drive_vel_cmd.left
+        self.drive_v_right = drive_vel_cmd.right
+
+    def _targetBearingCallback(self, channel, msg):
+        targetBearing = TargetBearing.decode(msg)
+        self.prev_target_bearing = self.target_bearing
+        self.target_bearing = targetBearing.target_bearing
+
+        target_diff = self.target_bearing - self.prev_target_bearing
+        self.target_bearing_rate = target_diff / \
```

Comment:
escape char?

Reply:
thats just from the python autoformatter but I'll get rid of it

Reply:
oh well ok thats fine then, if linter is happy im happy.